src/components/AppHeader/AppHeader.jsx

```diff
@@ -1,6 +1,15 @@
-// import Logo from "@/assets/spa-logo.png";
-// import "./AppHeader.scss";
-// const AppHeader = () => (
-// what goes in here? there is no content in the AppHeader component
-// );
-// export default AppHeader;
+import Logo from "@/assets/spa-logo.png";
+import "./AppHeader.scss";
+
+const AppHeader = () => {
+    return(
+        
+    <header className="app__header">
+        <img className="app__logo" src={Logo} alt="" />
```

Comment:
Works well! just alt description can be added.

Reply:
Description for the img alt added